pandas/io/json/json.py

```diff
@@ -281,14 +281,17 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=None,
         If True, infer dtypes; if a dict of column to dtype, then use those;
         if False, then don't infer dtypes at all, applies only to the data.
 
-        Not applicable with ``orient='table'``.
-
-        .. versionchanged:: 0.25
+        .. versionchanged:: 0.25.0
 
            Not applicable with ``orient='table'``.
 
     convert_axes : boolean, default True
```

Comment:
To match the signature you should update the docstring to say `None` here and make a comment within the note how `None` becomes `True` for all but table orients

Reply:
@WillAyd In order to be consistent, I was just following exactly the same convention as for the parameter `dtype`. Therefore I am going to change both `dtype` and `convert_axes`.